Tests/Acceptance/TokenEndpointTest.php

```diff
@@ -157,6 +157,38 @@ public function testSuccessfulAuthorizationCodeRequest(): void
         $this->assertNotEmpty($jsonResponse['access_token']);
     }
 
+    public function testSuccessfulAuthorizationCodeRequestWithPublicClient(): void
```

Comment:
@X-Coder264 Could you please add a test case for this code path?

https://github.com/thephpleague/oauth2-server/blob/8.0.0/src/Grant/AuthCodeGrant.php#L316

Reply:
@spideyfusion Sure. I added the test.